src/plugin/retail.js

```diff
@@ -0,0 +1,32 @@
+var logger = require('../utils/utils').logger;
+
+var preventKeysList = [];
+
+function shouldPreventKeyDefaultBehavior(key) {
+    if (!preventKeysList.length) return false;
+
+    return !!preventKeysList.find(function(preventKey) {
+        return preventKey.key === key;
+    });
```

Comment:
```suggestion
    return !!preventKeysList?.find(function(preventKey) {
        return preventKey.key === key;
    }) || false;
```
or if undefined is okay to return than this part `|| false` can be skipped

Reply:
I know but it will throw eslint error. Probably because `?` is not recognized as a valid keyword. That's why I didn't use optional chain keyword

Reply:
Keep it uniform, let, const and optional chaining aren't used in the project, and build will fail with current settings.